f3/granite.go

```diff
@@ -325,11 +327,14 @@ func (i *instance) isJustified(msg *GMessage) bool {
 }
 
 // Sends this node's QUALITY message and begins the QUALITY phase.
-func (i *instance) beginQuality() {
+func (i *instance) beginQuality() error {
```

Comment:
Discussion in #11 will have some bearing here. I am not keen on propagating error returns here for internal invariant problems. Maybe this will go away once the signing API is purely in terms of keys though?

Reply:
I think it came up for me with broadcast failing, which is not invariant problem.